src/uu/shred/src/shred.rs

```diff
@@ -462,6 +466,14 @@ fn wipe_file(
     if force {
         let metadata = fs::metadata(path).map_err_context(String::new)?;
         let mut perms = metadata.permissions();
+        #[cfg(target_family = "unix")]
+        // NOTE: set_readonly(false) makes the file world-writable on Unix.
+        #[allow(clippy::unnecessary_cast)]
+        // NOTE: S_IWUSR type is u16 on macOS.
+        perms.set_mode(perms.mode() | (S_IWUSR as u32));
+        #[cfg(not(target_family = "unix"))]
+        #[allow(clippy::permissions_set_readonly_false)]
+        // FIXME: Clippy still throws error on non-UNIX platforms.
```

Comment:
Is there a tracking issue for this false positive?

Reply:
I didn't find one.

Reply:
Thanks, could you reference it in the code too?